### PR TITLE
Gh 856 rest api execute single operation

### PR DESCRIPTION
**Pull request by GCHQDeveloper404**

### CODE REVIEW

File: rest-api/core-rest/src/main/java/uk/gov/gchq/gaffer/rest/service/IOperationService.java [hunk 1]
```diff
@@ -51,6 +52,11 @@
     Object execute(final OperationChain opChain);
 
     @POST
+    @Path("/operation")
```
Comment (t616178) [inline rest-api/core-rest/src/main/java/uk/gov/gchq/gaffer/rest/service/IOperationService.java]: Might it be better to change around the paths for these operations now we're adding the "execute single operation" option?

Something like:

Execute operation chain: /graph/doOperation/chain
Execute single operation: /graph/doOperation

?

File: rest-api/core-rest/src/main/java/uk/gov/gchq/gaffer/rest/service/IOperationService.java [hunk 2]
```diff
@@ -51,6 +52,11 @@
...
+    @ApiOperation(value = "Performs the given operation on the graph", response = Object.class)
+    Object execute(final Operation operation);
+
+    @POST
     @Path("/chunked")
```
Comment (t616178) [inline rest-api/core-rest/src/main/java/uk/gov/gchq/gaffer/rest/service/IOperationService.java]: For consistency, do we want to add a chunked implementation of the "execute single operation" endpoint?

**Comment by t616178** [comment]
Could some integration tests be added? This would probably involve adding some utility methods to the RestApiTestUtil class and then adding some new tests to the OperationTestIT class.

File: rest-api/core-rest/src/main/java/uk/gov/gchq/gaffer/rest/service/IOperationService.java (hunk 1 above)
Comment (p013570) [inline rest-api/core-rest/src/main/java/uk/gov/gchq/gaffer/rest/service/IOperationService.java]: I know there are a lot of changes in version 0.7.0 already, but that could be quite an annoying breaking change. It makes this tidier but doesn't really give any benefit. Perhaps something we think about doing at a later date.

**Pull request merged**